src/Goerli-DssSpell.t.sol

```diff
@@ -458,4 +508,347 @@ contract DssSpellTest is GoerliDssSpellTestBase {
+        scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done());
+
+        giveAuth(address(oracle), address(this));
+
+        (, address pip, , ) = oracle.ilks("RWA008-A");
+
+        assertEq(DSValueAbstract(pip).read(), bytes32(30_437_069 * WAD));
+
+        oracle.bump("RWA008-A", 40 * MILLION * WAD);
+
+        assertEq(DSValueAbstract(pip).read(), bytes32(40 * MILLION * WAD));
+    }
+
+    function testRWA008_INTEGRATION_TELL() public {
+        vote(address(spell));
+        scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done());
+
+        giveAuth(address(vat), address(this));
+        giveAuth(address(oracle), address(this));
+
+        (, , , uint48 tocPre) = oracle.ilks("RWA008-A");
+        assertTrue(tocPre == 0);
+        assertTrue(oracle.good("RWA008-A"));
+
+        vat.file("RWA008-A", "line", 0);
+        oracle.tell("RWA008-A");
+
+        (, , , uint48 tocPost) = oracle.ilks("RWA008-A");
+        assertTrue(tocPost > 0);
+        assertTrue(!oracle.good("RWA008-A"));
+    }
+
+    function testRWA008_INTEGRATION_TELL_CURE_GOOD() public {
+        vote(address(spell));
+        scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done());
+
+        giveAuth(address(vat), address(this));
+        giveAuth(address(oracle), address(this));
+
+        vat.file("RWA008-A", "line", 0);
+        oracle.tell("RWA008-A");
+
+        assertTrue(!oracle.good("RWA008-A"));
+
+        oracle.cure("RWA008-A");
+
+        assertTrue(oracle.good("RWA008-A"));
+        (, , , uint48 toc) = oracle.ilks("RWA008-A");
+        assertEq(uint256(toc), 0);
+    }
+
+    function testFailRWA008_INTEGRATION_CURE() public {
+        vote(address(spell));
+        scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done());
+
+        giveAuth(address(oracle), address(this));
+
+        oracle.cure("RWA008-A");
+    }
+
+    function testRWA008_INTEGRATION_TELL_CULL() public {
+        vote(address(spell));
+        scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done());
+
+        giveAuth(address(vat), address(this));
+        giveAuth(address(oracle), address(this));
+
+        assertTrue(oracle.good("RWA008-A"));
+
+        vat.file("RWA008-A", "line", 0);
+        oracle.tell("RWA008-A");
+
+        assertTrue(!oracle.good("RWA008-A"));
+
+        oracle.cull("RWA008-A", addr.addr("RWA008_A_URN"));
+
+        assertTrue(!oracle.good("RWA008-A"));
+        (, address pip, , ) = oracle.ilks("RWA008-A");
+        assertEq(DSValueAbstract(pip).read(), bytes32(0));
+    }
+
+    function testRWA008_OPERATOR_GET_RWA008_TOKEN() public {
+        assertEq(rwagem_008.balanceOf(rwaOperator_008), 1 * WAD);
+    }
+
+    function testRWA008_OPERATOR_LOCK_DRAW_CONDUITS_WIPE_FREE() public {
+        vote(address(spell));
+        scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done());
+
+        // TODO: Figure out why warping here makes this test fail
```

Comment:
Just calling this out to make sure we resolve it.  The warp was added because when rate is non-zero we all missed a bug in the urn that bricked it.  This test has to work with a warp/drip to test the rate accumulator.

Reply:
note, I can fix the error by setting line `680` to:
```solidity
uint256 daiToPay = (art * rate - dustInVat) / RAY + 2; // extra wei rounding
```

Reply:
Fixed.

Reply:
Solving this.